client/src/components/flex/components/flex_child.jsx

```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 
 export default styled.div`
-  flex: ${({ flex }) => flex}
+  ${({ flex }) => flex && `flex: ${flex};`};
```

Comment:
For default flex. I can change it back and be explicit with the `flex="0"` though!